content/en/01-about-pixie/04-faq.md

```diff
@@ -43,47 +45,73 @@ order: 4
+Pixie has been tested to work with a variety of Kubernetes environments listed in the [requirements](/installing-pixie/requirements/) page. Just because Pixie hasn’t been tested on a specific environment, does not mean that Pixie won't work. If you’ve successfully deployed Pixie to an environment not currently listed, please [submit a PR](https://github.com/pixie-labs/pixie-docs/blob/main/content/en/02-installing-pixie/01-requirements.md) to update the page.
 
-#### Can I completely self host Pixie?
+### Can I completely self host Pixie?
 
 Yes. See the self-hosted [Install Guide](/installing-pixie/install-guides/self-hosted-pixie) to get started.
 
-#### Does Pixie offer a hosted cloud offering?
+### Does Pixie offer a hosted cloud offering?
 
-Yes. Pixie Community Cloud is a hosted version of Pixie. Pixie stores its data on the cluster, resulting in a very small cloud footprint which allows Pixie to offer the Community Cloud offering 100% free for all Pixie users. See the Community Cloud [Install Guide](/installing-pixie/install-guides/community-cloud-for-pixie) to get started.
+Yes. Pixie Community Cloud is a hosted version of Pixie. Pixie stores all data on the customer's cluster, resulting in a very small cloud footprint which allows Pixie to offer the Community Cloud offering 100% free for all Pixie users. See the Community Cloud [Install Guide](/installing-pixie/install-guides/community-cloud-for-pixie) to get started.
 
 ## Data Collection
 
-#### Where does Pixie store its data? How much data is stored?
+### Where does Pixie store its data?
 
-Pixie stores the data it collects on the cluster in-memory; data is not sent to any centralized backend outside of the cluster.
+Pixie stores the data it collects in-memory on the nodes in your cluster; no data is sent to a centralized backend outside of the cluster. This is true for both self-hosted Pixie and [Pixie Community Cloud](/about-pixie/faq#does-pixie-offer-a-hosted-cloud-offering).
+
+Pixie has a [2GiB memory requirement](/installing-pixie/requirements/#memory) per node. After installing Pixie, it is normal to see a temporary increase in memory usage of the `vizier-pem` pods as they begin to fill their data tables.
+
+### How much data does Pixie store?
 
 Retention time depends on the level of traffic in your cluster, but will generally be on the order of hours. Each data table in Pixie (e.g. `http_events`) has its own maximum size. These tables collect data until the maximum size is reached, and then begin dropping the oldest data.
 
-We recommend integrating with third-party observability tools (such as New Relic’s integration) to provide long-term retention.
+We recommend integrating with third-party observability tools (such as [New Relic’s integration](https://newrelic.com/platform/kubernetes-pixie)) to provide long-term retention.
+
+### How does Pixie secure its data?
```

Comment:
@aimichelle - Could you review this section? Specifically, the _Community Cloud for Pixie Security_ blurb at the bottom.